### ISSUE TITLE
Filter content when Chromecasting Youtube to a TV set

### ISSUE THREAD

**Issue by craniack** [richardfrost/AdvancedProfanityFilter#206]
#### :loudspeaker: When I Chromecast a purchased Youtube video offensive content is not filtered.
When I Chromecast a youtube movie to my TV, APF can't see the captions to filter them.

**Comment by richardfrost** [comment]
Yeah, unfortunately the way casting works it isn't possible for the filter to be involved. Right now the best way I know of to use the filter with a TV is by hooking up a computer/laptop directly to the TV. I'll keep an eye out though, and if something changes I'll be happy to add the functionality.

**Comment by craniack** [comment]
Hooking up a laptop is a great idea that I should totally thought of.
 Thank you for getting back to me.

**Comment by richardfrost** [comment]
You're welcome! Enjoy!

**Comment by richardfrost** [comment]
So I was playing around last around with casting my screen (not a video) last night, and I was very surprised at how well it worked.

1. Go to the website that has the desired video in Chrome
2. Click on the browser options button, and then `Cast...`
3. At the bottom of that popup, click `Sources > Cast desktop`
4. Click on the desired device (Smart TV, Chromecast, etc)
5. Click on the "Desktop" with the browser window you'd like to play
6. Play the video from your laptop/computer

It wasn't perfect.. but it was WAY better than it was the last time I tried it (6+months ago). From Netflix and Plex it seemed perfectly watchable to me. Experiences will definitely vary based on your computer, the network connection between the computer and the TV, etc.
